packages/forms/docs/stories/UniversalPicker.story.js

```diff
@@ -0,0 +1,414 @@
+    `};
+
+    ${(props) =>
+        props.isFocused &&
+        `
+        border: solid 1px ${props.palette.brand.light};
+        box-shadow: ${props.shadows.shadow_4}, ${props.shadows.brand};
+    `};
+
+    ${(props) =>
+        props.isSelected &&
+        `
+        border: solid 1px ${props.palette.brand.main};
+    `};
+
+    ${(props) =>
+        props.disabled &&
+        `
+        border: solid 1px ${props.palette.border.disabled};
+        box-shadow: 0 0 0 0 transparent;
+        background-color: ${props.palette.background.disabled};
+        cursor: not-allowed;
+    `};
+`;
+
+const getInitialBorder = (props) => {
+    if (props.error) {
+        return `2px solid ${props.palette.error.main}`;
+    }
+    return `1px solid ${props.palette.border.divider}`;
+};
+
+const getColor = (props) => {
+    if (props.error) {
+        return props.palette.error.main;
+    }
+    return props.palette.brand.main;
+};
+
+const getShadow = (props) => {
+    if (props.error) {
+        return props.shadows.error;
+    }
+    return props.shadows.brand;
+};
+
+const StyledRadio = attachThemeAttrs(styled.span)`
+    width: 20px;
+    height: 20px;
+    display: inline-block;
+    position: relative;
+    vertical-align: middle;
+    border: ${getInitialBorder};
+    border-radius: ${BORDER_RADIUS_2};
+    background: ${(props) => props.palette.background.main};
+    box-sizing: border-box;
+    padding: 0;
+
+    &::after {
+        content: '';
+        height: 12px;
+        width: 12px;
+        position: absolute;
+        top: 2px;
+        left: 2px;
+        border-radius: ${BORDER_RADIUS_2};
+        background: transparent;
+        box-sizing: border-box;
+    }
+
+    ${(props) =>
+        props.isSelected &&
+        `
+        background: ${props.palette.background.main};
+        border: 2px solid;
+        border-color: ${getColor(props)};
+
+        &::after {
+            background: ${getColor(props)};
+            box-sizing: border-box;
+        }
+    `};
+
+    ${(props) =>
+        props.isFocused &&
+        `
+        border: 2px solid;
+        border-color: ${getColor(props)};
+        box-shadow: ${getShadow(props)};
+    `};
+
+    ${(props) =>
+        props.disabled &&
+        `
+        background-color: ${props.palette.background.disabled};
+        border-color: ${props.palette.border.divider};
+
+        &::after {
+            background: ${props.palette.background.main};
+            box-sizing: border-box;
+        }
+    `};
+`;
+
+const RadioItem = (props) => {
+    const { state, children, disabled, error } = props;
+
+    return (
+        <StyledRadioItem {...state} disabled={disabled} error={error}>
+            <div>{children}</div>
+            <StyledRadio {...state} disabled={disabled} error={error} />
+        </StyledRadioItem>
+    );
+};
+
+const StyledRadioPicker = styled(UniversalPicker)`
+    width: 500px;
+
+    > div {
+        flex-direction: column;
+    }
```

Comment:
we need to find another way to do this, maybe another prop variant, I will discuss it with @maxxgreene

Reply:
We discussed it already, we will add a `direction` prop that will manage this:
direction='horizontal' | 'vertical'
default value: 'horizontal'